src/applications/pre-need/config/form.jsx

```diff
@@ -92,6 +96,26 @@ const {
 
 const nonRequiredFullName = omit('required', fullName);
 
+function MailingAddressStateTitle(props) {
+  const { elementPath } = props;
+  const data = useSelector(state => state.form.data || {});
+  const country = get(elementPath, data);
+  if (!environment.isProduction() && country === 'CAN') {
+    return 'Province';
+  }
+  return 'State or territory';
+}
+
```

Comment:
Unfortunately due to the nature of event driven programming this had to be 3 individual methods. Otherwise they all call at the same time and return the same value messing up the expected return value.